src/components/media-player/ha-media-player-browse.ts

```diff
@@ -246,6 +248,13 @@ export class HaMediaPlayerBrowse extends LitElement {
             ],
             replace: true,
           });
+        } else if (
+          UNAVAILABLE_STATES.includes(this.hass.states[this.entityId]?.state)
+        ) {
+          this._setError({
+            message: "Entity unavailable",
```

Comment:
Is that something we can translate?

Reply:
What is the `err` we get when this happens?

Reply:
Entity not found

Reply:
Can we not use that instead of checking the state?

Reply:
We do that now. The issue is that the entity is found its just unavailable.